### PR TITLE
Buttons spacing

### PR DESCRIPTION
**Pull request by annaola**
For me, login and logout look better now.
login
logout

### CODE REVIEW

**Comment by Fanderman** [comment]
<div>
<input id="code" name="code" type="password" class="form-control" size="10">               
</div>
Why put one line of code in <div>? Previously it wasn't there. 
The aesthetic results are highly subjective so perhaps everyone should say what they think about this? 
Also would be nice if you could include pictures of how it looked previously, thanks in advance.

File: izp/polls/templates/polls/poll_detail.html [hunk 1]
```diff
@@ -3,18 +3,20 @@
...
          <div class="col-sm-3">
-            <label for="code">
+            <label for="code" style="margin-top:10px">
                Kod dostępu:
             </label>
-            <input id="code" name="code" type="password" class="form-control" size="10">
-            <button class="btn btn-primary">Zaloguj</button>
+            <div>
+               <input id="code" name="code" type="password" class="form-control" size="10">               
+            </div>
```
Comment (Fanderman) [inline izp/polls/templates/polls/poll_detail.html]: What's the point of that div when it contains only 1 line of code? Previously it wasn't there.

Comment (annaola) [reply izp/polls/templates/polls/poll_detail.html]: Right, input can be put without div

**Comment by annaola** [comment]
After changes: zrzut ekranu z 2017-12-12 22-02-01

**Pull request merged**